lib/talents.json

```diff
@@ -371,6 +371,30 @@
             "detail": "1/round, when you hit with a Main Melee weapon, gain 1 Blademaster Die."
           }
         ],
+        "actions": [
+          {
+            "name": "Parry",
+            "activation": "Reaction",
+            "trigger": "You are hit by a melee attack",
+            "detail": "You gain Resistance to all damage, heat, and burn dealt by the attack"
+          },
+          {
+            "name": "Deflect",
+            "activation": "Reaction",
+            "trigger": "You are hit by a ranged attack",
+            "detail": "You may roll any number of Blademaster Dice, expending them: if you roll a 5+ on any of these dice, you gain Resistance to all damage, heat, and burn dealt by the attak."
+          },
+          {
+            "name": "Feint",
+            "activation": "Free",
+            "detail": "Choose an adjacent character: when moving, you ignore engagement and don't provoke reactions from your target until the start of your next turn."
```

Comment:
Another nitpick: The `'` character in `don't` is originally a `’` character in the main text (so `don’t`). Incredibly minor, but for consistency I recommend just copy/pasting the curly apostrophe from the main description to these details (Feint and Trip).